docs/app/api_spec.md

```diff
@@ -102,10 +105,81 @@ Note: we will defer making any of these required for now
 }
 ```
 
+## Endpoint: `POST /feedback`
+
+### Description
+
+This endpoint accepts feedback from a navigator and responds with the user_id, value, step_id and comment given for the feedback request.
+
+### Request Parameters
+
+**Headers**
+
+- **Authorization**: Bearer token for authenticating API access.
+
+**Parameters**
+
+- **user_id** _(string, optional)_: Unique identifier for the navigator within ImagineLA.
+- **comment** _(string, optional)_: The user's feedback comment for the LLM response.
+
+- **session_id** _(string, required)_: Unique identifier for the current session, to track prior messages in the conversation.
+- **response_id** _(string, required)_: Unique identifier for the chatbot response.
+- **is_positive** _(bool, required)_: If the chatbot response is helpful to the navigator or not.
```

Comment:
Since LiteralAI logs this as a `float`, wondering if we should make this a float. @KevinJBoyer thoughts?

Reply:
I'd keep it as a bool since we only plan to support thumbs up/ thumbs down. If the design requirements change in the future (and we decide to let users respond with some continuous value) then we can update the API.